packages/driver/cypress/e2e/e2e/origin/patches.cy.ts

```diff
@@ -398,13 +427,16 @@ describe('src/cross-origin/patches', { browser: '!webkit' }, () => {
             () => {
               cy.on('uncaught:exception', (err) => {
                 expect(err.message).to.contain('404')
-                expect(Cypress.backend).not.to.have.been.calledWithMatch('request:sent:with:credentials')
 
                 return false
               })
 
               cy.get(`[data-cy="trigger-xml-http-request-with-bad-options"]`).click()
             })
+
+          cy.then(() => {
+            expect(Cypress.backend).to.have.been.calledWithMatch('request:sent:with:credentials')
```

Comment:
the backend request will actually be called in this case (not a typo)

Reply:
should we change the test name, like `still emits credential status in the case absolute url can be parsed even though request results in 404`?

Reply:
updated